liquibase-integration-tests/src/test/java/liquibase/dbtest/h2/H2IntegrationTest.java

```diff
@@ -189,6 +190,22 @@ public void testRollbackWithoutContext() throws Exception {
         Assert.assertEquals(insertedValue.toString(), queryResult.get(1).get(colName));
     }
 
+    @Test
+    public void makeSureDbmsFilteredChangeIsNotDeployed() throws Exception {
+        clearDatabase();
+
+        Liquibase liquibase = createLiquibase("changelogs/h2/complete/sql.change.dbms.filtered.should.not.be.deployed.changelog.xml");
+        liquibase.update();
```

Comment:
@MalloD12 if `liquibase.update();` fails, test will also fail, right?

Reply:
I think it would depend on the failure it returns, if there is a validation error but it doesn't stop the execution of the update then it will try to run the rest of the tests, but if for some reason `update` execution is stopped by an exception test will fail as well.